SDL_net.c

```diff
@@ -3,10 +3,28 @@
 #ifdef __WINDOWS__
 #define WIN32_LEAN_AND_MEAN 1
 #include <windows.h>
```

Comment:
You can safely drop windows.h include here: if winsock2 is to be included, it must be included before windows.h, and winsock2.h already includes windows.h.

Also a `#define _WIN32_WINNT 0x0600` would be good here I think, because you are using vista+ features (as far as I can see.)